docs/application/native/guides/device/battery-monitor.md

```diff
@@ -0,0 +1,199 @@
+
+    ```
+    #include <battery_monitor.h>
+    ```
+
+<a name="appusage_get"></a>
+## Getting Battery Usage Information of Application
+
+To get the battery usage information related with application ID:
+
+- You can get information about an application for a particular resource over a specific duration of time.
+  To get resource ID, use the `battery_monitor_resource_id_e` enum and to get the time duration type information, use the `battery_monitor_duration_type_e` enum:
+
+    ```
+    #include <battery_monitor.h>
+
+    void
+    func(void)
+    {
+       char* app_id = "org.tizen.samsung";
+
+       int error_code = BATTERY_MONITOR_ERROR_NONE;
+       int battery_usage = -1;
+       battery_monitor_resource_id_e resource_id = BATTERY_MONITOR_RESOURCE_ID_DISPLAY;
+       battery_monitor_duration_type_e duration_val = BATTERY_MONITOR_DURATION_TYPE_1DAY;
+
+       error_code = battery_monitor_get_usage_by_app_id_for_resource_id(app_id, resource_id, duration_val, &battery_usage);
+
+       if (error_code == BATTERY_MONITOR_ERROR_NONE)
+           printf("The Battery Usage for appid [%s], for resource display is [%d]", app_id, battery_usage);
+       else
+           printf("Error Occurred [%d]", error_code);
+
+       resource_id = BATTERY_MONITOR_RESOURCE_ID_WIFI;
+       duration_val = BATTERY_MONITOR_DURATION_TYPE_1WEEK;
+
+       error_code = battery_monitor_get_usage_by_app_id_for_resource_id(app_id, resource_id, duration_val, &battery_usage);
+
+       if (error_code == BATTERY_MONITOR_ERROR_NONE)
+           printf("The Battery Usage for appid [%s], for resource wifi is [%d]", app_id, battery_usage);
+
+       else
+           printf("Error Occurred [%d]", error_code);
+
+       free(app_id);
+
+       return;
+    }
+    ```
+
+- You can get information on total battery usage of an application ID by combining all the resources over a specific duration of time.
+  To get a particular duration type, use the `battery_monitor_duration_type_e` enum:
+
+    ```
+    duration_val = BATTERY_MONITOR_DURATION_TYPE_1DAY;
+    error_code = battery_monitor_get_total_usage_by_app_id(app_id, duration_val, &battery_usage);
+    if (error_code == BATTERY_MONITOR_ERROR_NONE)
+        printf("The total battery usage for appid [%s] over last 24 hrs is [%d]", app_id, battery_usage);
+    else
+        printf("Error Occurred [%d]", error_code);
+
+    duration_val = BATTERY_MONITOR_DURATION_TYPE_1WEEK;
+    error_code = battery_monitor_get_total_usage_by_app_id(app_id, duration_val, &battery_usage);
+    if (error_code == BATTERY_MONITOR_ERROR_NONE)
+        printf("The total battery usage for appid [%s] over last 7 days is [%d]", app_id, battery_usage);
+    else
+        printf("Error Occurred [%d]", error_code);
+
+    ```
+
+- You can get the battery usage values for all the resources used by an application ID over a specific duration of time.
```

Comment:
Thanks @mijin85cho
Can I change this sentence to:

Get battery usage information of an application ID for all resources over a specific duration of time.

As it conveys the API functioning more effectively.
"total" word is not correct in the context of the API. 
all the resources == > means that you can get information for all resources.
if one chooses add them together then it becomes total battery usage, otherwise its just battery usage.

Reply:
@d1-agarwal 
As suggested by @mijin85cho in 
[Comment 1](https://github.com/Samsung/tizen-docs/pull/847/files#r334313544)
[Comment 2](https://github.com/Samsung/tizen-docs/pull/847/files#r334313137)

I have rephrased it to the following final form: 

Kindly change lines 102-103:

- Get the battery usage information of an application ID  for all the resources over a specific duration of time.

   To get the resource ID, use the `battery_monitor_resource_id_e enum` and to get the duration type information, use the `battery_monitor_duration_type_e` enum:

Reply:
@d1-agarwal Is so, how about below?

Get each battery usage information of an application ID for every resources over a specific duration of time.

If I understood something wrong, please fix it.

Reply:
@d1-agarwal 
For me, the following sentence says that there are many batteries as **each** is used, which does not comply with the intentional meaning of the sentence.
 
Get each battery usage information of an application ID for every resources over a specific duration of time.
I had suggested the following: 

Get the battery usage information of an application ID for all the resources over a specific duration of time.

Let me know your opinion.

Reply:
@mijin85cho  @nawab-samsung 
Get the battery usage information of an application ID for each resource over a specific duration of time.